contracts/mocks/IAMB.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity 0.4.24;
+
+interface IAMB {
+    function messageSender() external view returns (address);
+    function transactionHash() external view returns (bytes32);
+    function withdrawFromDeposit(address _recipient) external;
+    function requireToPassMessage(address _contract, bytes _data, uint256 _gas) public;
```

Comment:
We need to leave only the methods that work in the `subsidized` mode.

Reply:
Thanks. Fixed https://github.com/poanetwork/poa-bridge-contracts/pull/77/commits/2655b4dbdabf0f6dd99ec074e50db3e5628ec719